components/Button.tsx

```diff
@@ -21,19 +22,21 @@ const Button: FunctionComponent<ButtonProps> = ({
   small,
   tooltipMessage = '',
   style,
+  type = 'button',
   ...props
 }) => {
   return (
     <button
       className={`${className} default-transition font-bold px-4 rounded-full ${
-        small ? 'py-1' : 'py-2.5'
+        small ? 'py-2' : 'py-2.5'
```

Comment:
messing with style of core components makes me nervous, can you comment? How can we be confident it doesn't mess stuff up elsewhere?